examples/atlas.rs

```diff
@@ -20,7 +20,7 @@ fn main() {
 		.init_resource::<TileHandles>()
 		.init_resource::<MyAtlas>()
 		.add_state(AppState::LoadTileset)
-		.add_system_set(SystemSet::on_enter(AppState::LoadTileset).with_system(load_tiles))
+		.add_system_set(SystemSet::on_update(AppState::LoadTileset).with_system(load_tiles))
```

Comment:
Rather than changing this, can you instead change this line from `set` to `overwrite_set`?https://github.com/MrGVSV/bevy_tile_atlas/blob/main/examples/atlas.rs#L57-L57

The result is pretty much the same either way, but loading the tileset on enter feels more idiomatic.